### PR TITLE
General improvements and bug fixes

### PR DESCRIPTION
**Pull request by digaru19**

### CODE REVIEW

**Comment by achilleas-k** [comment]
Is there a way to toggle the `DEBUG` flag externally or can it only be done in code for now?

**Comment by jgrewe** [comment]
maybe use argparse package to define the user interface

File: odmlui/dnd/tree.py [hunk 1]
```diff
@@ -21,11 +21,11 @@ def move(self):
 
     @property
     def copy(self):
-        return (self.action & gtk.gdk.ACTION_COPY) != 0
+        return (self.action == gtk.gdk.ACTION_COPY) != 0
```
Comment (achilleas-k) [inline odmlui/dnd/tree.py]: Clean up `!= 0`

File: odmlui/dnd/tree.py [hunk 2]
```diff
@@ -21,11 +21,11 @@ def move(self):
...
 
     @property
     def link(self):
-        return (self.action & gtk.gdk.ACTION_LINK) != 0
+        return (self.action == gtk.gdk.ACTION_LINK) != 0
```
Comment (achilleas-k) [inline odmlui/dnd/tree.py]: Clean up `!= 0`

**Review by achilleas-k — CHANGES_REQUESTED** [review]
Some comments:
- Clean up the two `!= 0` for the action comparisons.
- Add a command line debug flag that sets `DEBUG = True`.
- Print errors to `stderr` and debug output to `stdout`.

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**